core/storage/question/gae_models.py

```diff
@@ -233,6 +233,16 @@ def get_question_ids_linked_to_skill_ids(
 
     @classmethod
     def get_all_question_ids_linked_to_skill_id(cls, skill_id):
+        """Returns the list of all questions ids of a particular skill id.
+
+        Args:
+            skill_id: str. ID of the skill.
+
+        Returns:
+            list(question_id)|None. The list of question ids
+	    that are linked to a particular skill ID. None if the skill
```

Comment:
Please indent this line by 4 more spaces. Similarly for all other lines in continuation.

Reply:
Also, remove `question_id` from `list(question_id)`. This is the data type so, you should instead mention the data type of `question_id`.